lib/oref0-setup/bg_targets_raw.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Why is bg_targets_raw.json included? Can't find a reference of it being used.

Reply:
This is for x12 pumps: see #665